packages/common/src/projects/HubProject.ts

```diff
@@ -133,6 +137,24 @@ export class HubProject
     return getEntityEditorSchemas(i18nScope, type, options);
   }
 
+  /**
+   * Convert the project entity into a card view model that can
+   * be consumed by the suite of hug gallery components
+   *
+   * @param project project entity
+   * @param context auth & portal information
+   * @param target card link contextual target
+   * @param locale internationalization locale
+   */
+  static convertToCardViewModel(
```

Comment:
Do we really need to have the static method? I'm wondering if we should just use the instance method and require users to leverage `Project.fromJson()` if they have an `IHubProject` instead.

Reply:
This is a good question, and one I'd defer to @dbouwman.